docs/deployment-guide.md

```diff
@@ -255,7 +256,8 @@ the `scripts/docker` folder that is created when the system is built.
 
 Note, if you do not set the property `sleeper.retain.infra.after.destroy` to false when deploying then however you
 choose to tear down Sleeper later on you will also need to destroy some further S3 buckets and DynamoDB tables manually.
-This is because by default they are kept.
+This is because by default they are kept.  Similarly, by default all log groups will be retained when the Sleeper instance is
+torn down. You can set sleeper.retain.logs.after.destroy to false as well.
```

Comment:
The property name here should probably be quoted with backticks to format it as code, similar to the other property names.